### ISSUE TITLE
Incorrect handling of optional positional-only arguments with **kwargs

### ISSUE THREAD

**Issue by decorator-factory** [microsoft/pyright#892]
I am using Pyright v1.1.58 in VS Code.

Code:
```py
class A:
    def f(self, g: bool = False, /, **kwargs) -> None:
        ...

a = A()

a.f(hello="world")
```

Pyright doesn't like the `a.f(hello="world")` line in the following way: [see screenshot]
![image](https://user-images.githubusercontent.com/42166884/88864389-bfa6aa00-d20d-11ea-9f6b-feb271ab9b0f.png)

it incorrectly identifies that the first argument must be a bool and must be positional. Adding annotations to `kwargs` doesn't solve the issue.

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version of Pyright.

**Comment by erictraut** [comment]
This is addressed in Pyright 1.1.59, which I just published. It will also be included in the next release of Pylance.
